lib/api-call.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  getResponses: function(baseUrl, surveyID, endPoint, apiToken, apiTokenSecret) {
+    axios.get(baseUrl + '/' + surveyID + '/' + endPoint + '?api_token=' + apiToken + '&api_token_secret=' + apiTokenSecret)
```

Comment:
I would prefer this to be done in es6 format `${baseURL}/${surveyID}...` but it complained when attempting to do so

I would be grateful for ideas to make this look prettier